web/less/landing.less

```diff
@@ -170,6 +171,7 @@ a {
       text-shadow: black 1px 1px 12px;
     }
 
+    .room-name-input-container.visited,
```

Comment:
When `autoGenerateRoomName` is set to `false`, in landscape mobile the user name text field is hidden (or offscreen). (Testing in the Chrome mobile emulator.)